? DEADJOE
Index: fvwm/menustyle.c
===================================================================
RCS file: /home/cvs/fvwm/fvwm/fvwm/menustyle.c,v
retrieving revision 1.18
diff -u -3 -p -r1.18 menustyle.c
--- fvwm/menustyle.c	2002/10/21 22:10:52	1.18
+++ fvwm/menustyle.c	2002/10/22 04:44:07
@@ -539,6 +539,11 @@ void menustyle_update(MenuStyle *ms)
 		c_active.hilight = ST_MENU_RELIEF_COLORS(ms).fore;
 		c_active.shadow = ST_MENU_RELIEF_COLORS(ms).back;
 	}
+	if (ST_USING_DEFAULT_FONT(ms))
+	{
+		ST_PSTDFONT(ms) = Scr.DefaultFont;
+	}
+
 	if (ST_HAS_GREYED_CSET(ms))
 	{
 		c_stipple.fore = greyed_cs->fg;

