*** move_resize.c.orig	Wed Jul 31 10:52:59 2002
--- move_resize.c	Wed Jul 31 10:59:36 2002
***************
*** 2969,2974 ****
--- 2969,2977 ----
        get_outline_rects(
  	&(rects[1]), move_resize_grid.geom.x, move_resize_grid.geom.y,
  	move_resize_grid.geom.width, move_resize_grid.geom.height);
+ fprintf(stderr,"dmrg: erasing (%d) %4d %4d %4dx%4d\n", nrects,
+ 	move_resize_grid.geom.x, move_resize_grid.geom.y,
+ 	move_resize_grid.geom.width, move_resize_grid.geom.height);
    }
    if (width && height)
    {
***************
*** 2978,2983 ****
--- 2981,2988 ----
      move_resize_grid.geom.width = width;
      move_resize_grid.geom.height = height;
      nrects += get_outline_rects(&(rects[0]), x, y, width, height);
+ fprintf(stderr,"dmrg: drawing (%d) %4d %4d %4dx%4d\n",
+ 	nrects, x, y, width, height);
    }
    if (nrects > 0)
    {

