? fvwm-2.5.9
Index: debian/Makefile.am
===================================================================
RCS file: /home/cvs/fvwm/fvwm/debian/Makefile.am,v
retrieving revision 1.4
diff -u -r1.4 Makefile.am
--- debian/Makefile.am	14 Nov 2003 18:55:15 -0000	1.4
+++ debian/Makefile.am	13 Mar 2004 11:35:19 -0000
@@ -45,7 +45,7 @@
 	echo "" >> debian/changelog
 	echo "  * new upstream release" >> debian/changelog
 	echo "" >> debian/changelog
-	echo " -- $(fullname) <$(email)>  $(datum)" >> debian/changelog
+	(LANG=en_UK && LC_ALL=en_UK && echo " -- $(fullname) <$(email)>  $(datum)" >> debian/changelog)
 	./configure --prefix=/usr --mandir=/usr/share/man --libexecdir=/usr/lib --sysconfdir=/etc/X11/fvwm $(cparams)
 	fakeroot dh_testdir -P$(fullinstdir)
 	fakeroot dh_testroot -P$(fullinstdir)

