*** module_interface.c	Mon Apr 29 03:40:57 2002
--- module_interface.c.orig	Mon Apr 29 03:25:24 2002
***************
*** 1101,1108 ****
    *len =
      FvwmPacketHeaderSize + num + (strlen(name) / sizeof(unsigned long)) + 1;
    /* truncate long packets */
!   if (*len > FvwmPacketMaxSize)
!     *len = FvwmPacketMaxSize;
  
    body = (unsigned long *)safemalloc(*len * sizeof(unsigned long));
    body[*len-1] = 0; /* Zero out end of memory to avoid uninit memory access. */
--- 1101,1108 ----
    *len =
      FvwmPacketHeaderSize + num + (strlen(name) / sizeof(unsigned long)) + 1;
    /* truncate long packets */
!   if (*len > FvwmPacketMaxSize_byte)
!     *len = FvwmPacketMaxSize_byte;
  
    body = (unsigned long *)safemalloc(*len * sizeof(unsigned long));
    body[*len-1] = 0; /* Zero out end of memory to avoid uninit memory access. */
