--- parse.c.orig	Tue Oct 15 12:27:13 2002
+++ parse.c	Tue Oct 15 12:26:58 2002
@@ -1546,7 +1546,6 @@
   unsigned int height;
 
   flags = FScreenParseGeometry(geom,&g_x,&g_y,&width,&height);
-fprintf(stderr,"fb: parsing geometry %d %d %dx%d\n", g_x, g_y, width, height);
   UberButton->w = 0;
   UberButton->h = 0;
   UberButton->x = 0;
@@ -1615,7 +1614,6 @@
       break;
     case 4:
       /* Xinerama state */
-fprintf(stderr,"fb: configuring xinerama: %s\n", rest);
       FScreenConfigureModule(rest);
       break;
     }

