--- /usr/lib/fvwm/2.5.11/FvwmTabs	2004-07-19 23:32:28.000000000 +0300
+++ tmp/fvwmtabs/FvwmTabs-v3.3/FvwmTabs	2004-07-15 16:11:19.000000000 +0300
@@ -406,7 +406,7 @@
 {
 	my $tId = shift();
 
-	if (!defined $tId or $tId eq '#')
+	if (!defined $tId)
 	{
 		for ($tId = 0; ; $tId++)
 		{
@@ -424,7 +424,7 @@
 		return;
 	}
 
-	my $title = $fvwm->name() . "[$tId]";
+	my $title = $fvwm->name() . " [$tId]";
 	my $tl = $TOP->Toplevel(Name => $fvwm->name(),
 							-class => $fvwm->name(),
 							-title => $title);

