Index: focus.c
===================================================================
RCS file: /home/cvs/fvwm/fvwm/fvwm/focus.c,v
retrieving revision 1.121
diff -u -r1.121 focus.c
--- focus.c	2002/08/23 02:31:37	1.121
+++ focus.c	2002/10/06 01:44:23
@@ -302,7 +302,7 @@
 	{
 		return;
 	}
-	__update_windowlist(fw, args->is_focus_by_focus_cmd, args->set_by);
+	__update_windowlist(fw, args->set_by, args->is_focus_by_focus_cmd);
 	if (__try_other_screen_focus(fw) == True)
 	{
 		return;

